WordPress/Classes/System/WPGUIConstants.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+extern const CGFloat WPAlphaFull;
+extern const CGFloat WPAlphaZero;
+
+extern const CGFloat WPColorFull;
+extern const CGFloat WPColorZero;
+
+extern const CGFloat WPAnimationDurationDefault;
+extern const CGFloat WPAnimationDurationFast;
+extern const CGFloat WPAnimationDurationFaster;
```

Comment:
:+1: for putting this in a separate file. There are probably other consts we can bring here from other parts of the app.